Added obsidian-hd-unfilled-stats-highlighter to community-plugins.json.

White7292 opened the pull request:
# I am submitting a new Community Plugin

## Repo URL

<!--- Paste a link to your repo here for easy access -->
Link to my plugin: https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

joethei commented:
[## Releasing new releases](https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter/blob/d79270eeb6e81e45181c85691e656ba0647a58e8/README.md#L21-L63)
Remove this, it will only confuse users.

[this.registerDomEvent(document, "keyup", (evt: KeyboardEvent) => {](https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter/blob/d79270eeb6e81e45181c85691e656ba0647a58e8/main.ts#L43)
use the ['editor-change`](https://github.com/obsidianmd/obsidian-api/blob/bceb489fc25ceba5973119d6e57759d64850f90d/obsidian.d.ts#L4239-L4243) event instead.

White7292 commented:
> [## Releasing new releases](https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter/blob/d79270eeb6e81e45181c85691e656ba0647a58e8/README.md#L21-L63) Remove this, it will only confuse users.
> 
> [this.registerDomEvent(document, "keyup", (evt: KeyboardEvent) => {](https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter/blob/d79270eeb6e81e45181c85691e656ba0647a58e8/main.ts#L43) use the ['editor-change`](https://github.com/obsidianmd/obsidian-api/blob/bceb489fc25ceba5973119d6e57759d64850f90d/obsidian.d.ts#L4239-L4243) event instead.

Hey @joethei! 
Thanks for the quick review.

I've made the requested changes: 
- Removed the `Releasing new releases` section of the README.md.
- Registered `editor-change` event instead of the `keyup` event.
- Updated the Release 1.0.0 with the new JS file.

liamcain commented:
- [getActiveViewOfType](https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter/blob/4e678e6e605f05fb9b06e1d9b69e9344f9c04593/main.ts#L56) you should use [`activeEditor`](https://github.com/obsidianmd/obsidian-api/blob/38dd22168d2925086371bfc59e36fd9121527a39/obsidian.d.ts#L4009) instead so that your plugin works within Canvas
- [Adding](https://github.com/White7292/obsidian-hd-unfilled-stats-highlighter/blob/4e678e6e605f05fb9b06e1d9b69e9344f9c04593/README.md#L21) remove all the unrelated info from the README
